src/scripts/main.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+thumbs.addEventListener('click', (e) => {
+  e.preventDefault();
```

Comment:
The call to `e.preventDefault()` is correct if you want to prevent the default action of the anchor tag. However, ensure that this is the desired behavior, as it will prevent the browser from navigating to the link's href.